splash/browser_tab.py

```diff
@@ -537,6 +537,13 @@ def http_get(self, url, callback, headers=None, follow_redirects=True):
             follow_redirects=follow_redirects
         )
 
+    def http_post(self, url, callback, headers=None, follow_redirects=True, body=None):
```

Comment:
What happens if `follow_redirects` is True (default) and a response for our POST request is a redirect? We shouldn't send the same POST response again to the new location, it should become a GET request. See https://github.com/scrapy/scrapy/blob/181f8d27d782cb433de5f78770ee1446c62991cd/scrapy/downloadermiddlewares/redirect.py#L54